tests/test_autochunk/test_autochunk_openfold_codegen.py

```diff
@@ -0,0 +1,160 @@
+from functools import partial
+
+import pytest
+import torch
+import torch.fx
+import torch.multiprocessing as mp
+
+import colossalai
+from colossalai.core import global_context as gpc
+from colossalai.fx._compatibility import is_compatible_with_meta
+from colossalai.fx.codegen.activation_checkpoint_codegen import CODEGEN_AVAILABLE
+from colossalai.fx.graph_module import ColoGraphModule
+from colossalai.fx.passes.meta_info_prop import MetaInfoProp
+from colossalai.fx.tracer.experimental import ColoTracer, symbolic_trace
+from colossalai.utils import free_port
+from tests.test_autochunk.evoformer.evoformer import evoformer_base
+from tests.test_autochunk.origin_openfold.evoformer import EvoformerBlock
```

Comment:
can we import EvoformerBlock from a 3rd party software? I believe including an original openfold implementation in the test directory is not a good idea.

Reply:
OK, next PR is fine.

Reply:
Already done now.